Gemfile.lock

```diff
@@ -0,0 +1 @@
+
```

Comment:
remove this line

Reply:
made all the changes please review my PR @AvikantSrivastava  bhaiya